packages/experimental/Shadow/src/__tests__/__snapshots__/Shadow.test.tsx.snap

```diff
@@ -1,10 +1,408 @@
+    </Text>
+  </View>
+</View>
+`;
+
+exports[`Shadow component tests Brand shadow (depth=4) 1`] = `
+<View
+  style={
+    Object {
+      "shadowColor": "#000000",
+      "shadowOffset": Object {
+        "height": 0,
+        "width": 0,
+      },
+      "shadowOpacity": 0.3,
+      "shadowRadius": 2,
+    }
+  }
+>
+  <View
+    style={
+      Object {
+        "shadowColor": "#000000",
+        "shadowOffset": Object {
+          "height": 2,
+          "width": 0,
+        },
+        "shadowOpacity": 0.25,
+        "shadowRadius": 4,
+      }
+    }
+  >
+    <Text>
+      Brand shadow (depth=4)
+    </Text>
+  </View>
+</View>
+`;
+
+exports[`Shadow component tests Brand shadow (depth=8) 1`] = `
+<View
+  style={
+    Object {
+      "shadowColor": "#000000",
+      "shadowOffset": Object {
+        "height": 0,
+        "width": 0,
+      },
+      "shadowOpacity": 0.3,
+      "shadowRadius": 2,
+    }
+  }
+>
+  <View
+    style={
+      Object {
+        "shadowColor": "#000000",
+        "shadowOffset": Object {
+          "height": 4,
+          "width": 0,
+        },
+        "shadowOpacity": 0.25,
+        "shadowRadius": 8,
+      }
+    }
+  >
+    <Text>
+      Brand shadow (depth=8)
+    </Text>
+  </View>
+</View>
+`;
+
+exports[`Shadow component tests Brand shadow (depth=16) 1`] = `
+<View
+  style={
+    Object {
+      "shadowColor": "#000000",
+      "shadowOffset": Object {
+        "height": 0,
+        "width": 0,
+      },
+      "shadowOpacity": 0.3,
+      "shadowRadius": 2,
+    }
+  }
+>
+  <View
+    style={
+      Object {
+        "shadowColor": "#000000",
+        "shadowOffset": Object {
+          "height": 8,
+          "width": 0,
+        },
+        "shadowOpacity": 0.25,
+        "shadowRadius": 16,
+      }
+    }
+  >
+    <Text>
+      Brand shadow (depth=16)
+    </Text>
+  </View>
+</View>
+`;
+
+exports[`Shadow component tests Brand shadow (depth=28) 1`] = `
+<View
+  style={
+    Object {
+      "shadowColor": "#000000",
+      "shadowOffset": Object {
+        "height": 0,
+        "width": 0,
+      },
+      "shadowOpacity": 0.3,
+      "shadowRadius": 8,
+    }
+  }
+>
+  <View
+    style={
+      Object {
+        "shadowColor": "#000000",
+        "shadowOffset": Object {
+          "height": 14,
+          "width": 0,
+        },
+        "shadowOpacity": 0.25,
+        "shadowRadius": 28,
+      }
+    }
+  >
+    <Text>
+      Brand shadow (depth=28)
+    </Text>
+  </View>
+</View>
+`;
+
+exports[`Shadow component tests Brand shadow (depth=64) 1`] = `
+<View
+  style={
+    Object {
+      "shadowColor": "#000000",
+      "shadowOffset": Object {
+        "height": 0,
+        "width": 0,
+      },
+      "shadowOpacity": 0.3,
+      "shadowRadius": 8,
+    }
+  }
+>
+  <View
+    style={
+      Object {
+        "shadowColor": "#000000",
+        "shadowOffset": Object {
+          "height": 32,
+          "width": 0,
+        },
+        "shadowOpacity": 0.25,
```

Comment:
Nit: This is supposed to be 24% but is rounded to 25%. In practice I don't think the 1% difference matters but wanted to call it out.

Reply:
Ah thanks for catching that, I didn't notice. Yeah I'm not sure what I can do about this, I doubt there's a perfect way to extract the right value. Maybe I can note it down as a known issue in the shadowStyleFromTheme method